run.py

```diff
@@ -248,11 +248,12 @@ def process(
         # n_workers = multiprocessing.cpu_count()  # FIXME should use N_JOBS
         n_workers = 4
         logger.info(f'👾 Initializing Dask client with {n_workers} workers …')
+        dask_temp_dir = "./.dask-worker-space"
```

Comment:
Careful with this, as this will spill memory to disk in the current working directory, which could leads to problems

Reply:
this is gone now